DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/DependencyInjection/AutofacArcGISModule.cs

```diff
@@ -55,6 +66,10 @@ protected override void Load(ContainerBuilder builder)
       .RegisterType<ScopedFactory<ISpeckleConverterToSpeckle>>()
```

Comment:
this ScopedFactory<> has been replaced with the UoW, these types will not be wrapped in the unit of work but it will be the Send and Recieve operations, probably these should be InstancePerLifetimeScope() still